### PR TITLE
Make names in sync formalism more memorable

### PR DESCRIPTION
**Pull request by krOoze**

### CODE REVIEW

**Comment by ikapoura** [comment]
I think that MemOps is more fitting than OpsMem because it fits in how we phrase it in our heads as "memory operations". You also introduce MemDep for "memory dependency" and we would have an inconsistency there. So, we could end with MemOps and ScopedMemOps.

**Comment by krOoze** [comment]
I am not sure memory accesses are "operations" in Vulkan. That would be potentially misleading, because memory is not subject to execution dependency as described in the first formalism bullet list.

File: chapters/synchronization.txt [hunk 1]
```diff
@@ -70,32 +70,37 @@ If an operation happens-before another operation, then the first operation
...
 chain can be described by substituting the following in the description of
 execution dependencies:
 
-  * Let *S* be a set of synchronization commands that generate an execution
-    dependency chain.
-  * Let *A~S~* be the first synchronization scope of the first command in
-    *S*.
-  * Let *B~S~* be the second synchronization scope of the last command in
-    *S*.
+  * Let *SyncChain* be a set of synchronization commands that generate an
```
Comment (oddhack) [inline chapters/synchronization.txt:98]: I don't think this is quite right - it is saying to substitute these terms in the language above, but there \*Sync* is referred to, not \*SyncChain*. It almost works except for the "Submitting \*Ops~1~*, \*Sync* and \*Ops~2~* for execution," clause. Using the same label for \*Sync* and \*SyncChain* would address this.

**Review by oddhack — CHANGES_REQUESTED** [review]
Seems fine once the \*Sync* / \*SyncChain* issue I noted is addressed.

File: chapters/synchronization.txt (hunk 1 above)
Comment (krOoze) [reply chapters/synchronization.txt]: Ok, sticking with the original "Sync" name for now.

File: chapters/synchronization.txt [hunk 2]
```diff
@@ -151,18 +156,25 @@ and execution order between two sets of operations.
...
+    *ScopedOps~2~*.
+  * Let *AccessScope~1st~* be the first access scope of the first command in
+    *SyncChain*.
+  * Let *AccessScope~2nd~* be the second access scope of the last command in
+    *SyncChain*.
+  * Let *ScopedOpsMem~1~* be the intersection of sets *OpsMem~1~* and
+    *AccessScope~1st~*.
+  * Let *ScopedOpsMem~2~* be the intersection of sets *OpsMem~2~* and
+    *AccessScope~2nd~*.
+  * Submitting *Ops~1~*, *SyncChain*, and *Ops~2~* for execution, in that
```
[the remark this answers is not in the document]
Comment (oddhack) [reply chapters/synchronization.txt]: This appears done but not marked auto-resolved by GH for some reason. LGTM now.

**Review by oddhack — APPROVED** [review]

**Review by oddhack — APPROVED** [review]

**Pull request merged**